File: letter/ru/Р.py
from common import *

expr = Eq(
    div(
        (x + S(1) / 2)
        * (S(1) / 4 * (x + S(1) / 2) ** 2 + (y - S(1) / 2) ** 2 - S(1) / 4)
        * sqrt(x + S(1) / 2 + eps),
        sqrt(1 + eps - Abs(y))
    )
    ,
    0
)
size = 1

File: letter/ru/Р.js
import { Letter } from '../../Letter.js';

const expr = '(x + 1/2) * (1/4*(x + 1/2)^2 + (y - 1/2)^2 - 1/4) * sqrt(x + 1/2 + 1/10) / sqrt(1 + 1/10 - abs(y))';

export const letter = new Letter(expr, 1);
